### ISSUE TITLE
Screen Flickering (Black) while playing Jak 2 on Tooling v0.2.8

### ISSUE THREAD

**Issue by Matty2024** [open-goal/jak-project#3406]
### Acknowledgements

- [X] I have [searched](https://github.com/open-goal/jak-project/issues?q=is%3Aissue+is%3Aopen+label%3Abug+label%3Ajak2+sort%3Aupdated-desc+) for duplicate or closed bug reports
- [X] I understand that I am supposed to provide my own legitimately obtained copy of the game

### Describe the Bug

After updating the OpenGoal launcher to Tooling v0.2.8 I'm getting terrible black screen flickering constantly while playing.

It's very uncomfortable to play with so I have downgraded to v0.2.7 again and all is fine. 
Not sure what has changed in the new version but Control Z please.

Imaging plugging in a external monitor to your laptop and your screen flickers a little while it mirrors, extends etc.
Now, this is happening constantly. I'm playing the game in "full screen mode" and on regular 30/60 fps.
 
I did try to screen recording my game-play but upon watching it back there were no issues in the video.
I wanted to report this early in case this isn't changed in the future because Jak 2 is "finished".

I hope this is enough assistance to find what might be causing it.

### How To Reproduce

Use Tooling v0.2.8 with Launcher v2.4.2 with Jak 2 in full screen mode.

### Does this problem occur on original hardware or PCSX2?

Yes, it's unique to OpenGOAL

### Expected Behavior

Shouldn't have black screen flickering. 

### Environment Information

[opengoal-support-package.zip](https://github.com/open-goal/jak-project/files/14513037/opengoal-support-package.zip)

### Game Version

PAL

### Have you set the game to something other than `60fps`?

No

**Comment by ManDude** [comment]
In your Windows display settings, what value is this set to? Also, does this occur in borderless windowed mode?
![image](https://github.com/open-goal/jak-project/assets/7569514/3e09b9f6-c9b1-4b14-b194-93ae28e6b1af)

**Comment by xTVaser** [comment]
I'm not aware of anything in 0.2.8 that would have effected this. One further question:
- Does it occur in jak 1 as well?

I have a branch that tries to cleanup and address the display mode stuff specifically, so I'm curious if that would solve your problem.  I'll try and get that bundled up in a way you can test via the launcher.

**Comment by Matty2024** [comment]
> In your Windows display settings, what value is this set to? Also, does this occur in borderless windowed mode? ![image](https://private-user-images.githubusercontent.com/7569514/310584749-3e09b9f6-c9b1-4b14-b194-93ae28e6b1af.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk3NTI2NDAsIm5iZiI6MTcwOTc1MjM0MCwicGF0aCI6Ii83NTY5NTE0LzMxMDU4NDc0OS0zZTA5YjlmNi1jOWIxLTRiMTQtYjE5NC05M2FlMjhlNmIxYWYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDZUMTkxMjIwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OWMzMWY4OGViZDExMDcwY2EyOGE0ZjRmNjM0MzMwMGIxMjBjNjlmNzdmYjg3YjgxY2VmYjc0ZDRkNzRhMGJhOSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.uop6swaCi-h9AsP2hF0QSLLO_9W516Vh_ez963S61QY)

Hey ManDude, thanks very much for the reply.

My laptop comes standard with "125% scaling (Recommended)" and I haven't changed that since I bought the laptop a year ago. Again, this issue is ONLY when running v0.2.8 tooling version and all previous versions have no black screen flickering. 

I will however, for science, tomorrow test the game in borderless as well.

**Comment by ManDude** [comment]
Please tell us if it still happens if you set it to 100% and then start the game.

**Comment by Matty2024** [comment]
> I'm not aware of anything in 0.2.8 that would have effected this. One further question:
> 
> * Does it occur in jak 1 as well?
> 
> I have a branch that tries to cleanup and address the display mode stuff specifically, so I'm curious if that would solve your problem. I'll try and get that bundled up in a way you can test via the launcher.

Hey xTVaser, 

Thanks very much for the reply and the assistance. I didn't actually check Jack 1. As soon as the issues where starting in Jak 2, I immediately switched back to the previous tooling version so I could play normally again..    

I will test this out for you tomorrow along with the request from Mandude and update accordingly. Really appreciate the assistance. :)

**Comment by Matty2024** [comment]
> Please tell us if it still happens if you set it to 100% and then start the game.

Update: (Decided to do it now quickly and postpone other things so I don't waste anyone's time)

I swapped the display scaling to 100% and it appears as if things are normal again. Thank you kindly for the suggestion.
I ran Jak 1 & Jack 2 with this scaling enabled and all are running normally with the latest tooling v0.2.8.

Maybe there was just some display glitch (on my side) while loading the game which caused all the issues and nothing was wrong with app itself. 

Thank you both for the assistance. Have a great day further.

**Comment by ManDude** [comment]
Does it occur in borderless windowed at 125%?

**Comment by xTVaser** [comment]
As usual I unfortunately can't reproduce this on my end.  However I did notice something potentially odd in your PC settings.

I assume you are using a 1920x1080 monitor.  Your pc-settings has this:
```
  (window-size 1920 1080)
  (game-size 1920 1080)
```

But at 125% scaling, your maximum resolution would be 1440x810

I'd be curious if you reduce those settings down if the flickering would go away.  The file is in `AppData/Roaming/OpenGOAL/jak2/pc-settings.gc`

**Comment by Matty2024** [comment]
> As usual I unfortunately can't reproduce this on my end. However I did notice something potentially odd in your PC settings.
> 
> I assume you are using a 1920x1080 monitor. Your pc-settings has this:
> 
> ```
>   (window-size 1920 1080)
>   (game-size 1920 1080)
> ```
> 
> But at 125% scaling, your maximum resolution would be 1440x810
> 
> I'd be curious if you reduce those settings down if the flickering would go away. The file is in `AppData/Roaming/OpenGOAL/jak2/pc-settings.gc`

Hey xTVaser & Mandue, 

Thank you for all your assistance with this and my apologies for the wasted time. 
I have tried to reproduce this bug today that occurred last night for me and I'm unable to.

It must have been a display glitch on my laptop but it just seemed too coincidental that when I closed the game, the flickering went away and then proceeded to come back again as soon as I launched Jak 2. Then downgrading the Tooling version fixed it without restarting my laptop or anything.

I will refrain from posting bugs in the future as it appears your code is always 100%! and it's simply a Id10T error on my side and I'm too quick to report an "issue" when its probably just a glitch that can be resolved with a restart.

My apologies again and heave a great day further.
